### ISSUE TITLE
Linux Disks iteration should exclude loopback and ram

### ISSUE THREAD

**Issue by dbwiddis** [dblock/oshi#148]
When running #147 on Ubuntu, 16 ram disks are displayed, along with 8 loopback disks.  This is rather spammy.  These should be filtered out.
```
Disks:
 /dev/sda: (model: Ubuntu_Linux-0 - S/N: XQ1PCFXNN6YMN4JDF120) reads (in bytes): 1199864832 writes (in bytes): 936181760 
 /dev/sr0: (model: Virtual_DVD-ROM__1_ - S/N: -_31415B265) reads (in bytes): 0 writes (in bytes): 0 
 /dev/sr1: (model: Virtual_DVD-ROM__2_ - S/N: -_31415C265) reads (in bytes): 0 writes (in bytes): 0 
 /dev/loop0: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/loop1: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/loop2: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/loop3: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/loop4: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/loop5: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/loop6: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/loop7: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram0: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram1: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram10: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram11: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram12: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram13: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram14: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram15: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram2: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram3: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram4: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram5: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram6: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram7: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram8: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
 /dev/ram9: (model: Unknown - S/N: Unknown) reads (in bytes): 0 writes (in bytes): 0 
[main] INFO oshi.SystemInfoTest - Checking Displays...
```
Also the disk enumeration is not included in the JSON; it needs to be added to the `AbstractHardwareAbstractionLayer`'s `toJSON()` method.

**Comment by dbwiddis** [comment]
Added the necessary JSON in #149.  Still need the loop and ram filtered out (or other approach to this issue, @henryx ).  Sample output in README would be useful too.

**Comment by dbwiddis** [comment]
Fixed in [e090a8f](https://github.com/dblock/oshi/commit/e090a8ff8796da5332ac443e1246efcd23536aad)
